### PR TITLE
Mech Stress Test Experiment is now SIGNIFICANTLY LESS ASS to do

### PR DESCRIPTION
**Pull request by DATA-xPUNGED**
## About The Pull Request

The Stress Test 1 experiment now only requires ONE mech to be broken down, and the margin of error of the percentage is much more lenient, being 5% up or down (it was previously only 2%).
## Why It's Good For The Game

This experiment is painful to do right now, the process usually involves making TWO mechs, damaging them *very close* to the arbitrary percentage the experiment requires, scan it, realize you went too low so you gotta weld it back up and start hitting it with your crowbar again, scan it, get the experiment, pick up the SECOND experiment, kill the mechs, scan THOSE. Everyone finds these a pain in the ass
## Changelog
:cl:
balance: Exosuit Materials 1 now only requires one mech.
qol: Exosuit Materials 1 is much more lenient on the percentage it requires.
/🆑

### CODE REVIEW

**Comment by zxaber** [comment]
I don't agree with the changes to the first experiment. It's really easy to do if you park the mech at a charging bay and use the computer to monitor the mech's damage. 

The second experiment is absolutely a pain in the ass, I had underestimated how lame it would be. But I do plan on replacing it soon with an experiment about scanning fully equipped not-mapload mechs, to encourage building mechs for other crew members.

**Comment by DATA-xPUNGED** [comment]
> I don't agree with the changes to the first experiment. It's really easy to do if you park the mech at a charging bay and use the computer to monitor the mech's damage.
> 
> The second experiment is absolutely a pain in the ass, I had underestimated how lame it would be. But I do plan on replacing it soon with an experiment about scanning fully equipped not-mapload mechs, to encourage building mechs for other crew members.

I wasn't aware of the charging bay thing, actually, but i still feel like being this close shouldn't be a requirement. I'm willing to cut the leniency in half  though

**Comment by D4C-420** [comment]
>I wasn't aware of the charging bay thing, actually, but i still feel like being this close shouldn't be a requirement. I'm willing to cut the leniency in half though

You can also just hop in the mech to see how much damage its taken.

**Comment by DATA-xPUNGED** [comment]
> > I wasn't aware of the charging bay thing, actually, but i still feel like being this close shouldn't be a requirement. I'm willing to cut the leniency in half though
> 
> You can also just hop in the mech to see how much damage its taken.

Having to repeatedly hop in is also annoying as shit

**Comment by DaydreamIQ** [comment]
Personally i've found it really easy to reach the margin. Just hit it with a welder until its roughly within like 10% of what you need it to be, then just use a screwdriver or crowbar to do less damage. Its kind of a skill issue if you don't use the mech pad that's RIGHT THERE TO DO IT WITH

**Comment by DATA-xPUNGED** [comment]
> Personally i've found it really easy to reach the margin. Just hit it with a welder until its roughly within like 10% of what you need it to be, then just use a screwdriver or crowbar to do less damage. Its kind of a skill issue if you don't use the mech pad that's RIGHT THERE TO DO IT WITH

The actual margin of error was 2% up or 2% down the desired percentage, i've made it 5% up and 5% down

**Comment by githubuser4141** [comment]
> > > I wasn't aware of the charging bay thing, actually, but i still feel like being this close shouldn't be a requirement. I'm willing to cut the leniency in half though
> > 
> > 
> > You can also just hop in the mech to see how much damage its taken.
> 
> Having to repeatedly hop in is also annoying as shit

You can also just guesstimate it. Hit it once, see the damage, memorize this forever, now you know. Don't bother hopping in, just do it x amount of times, examine it, memorize the flavor text examine messages (slightly, heavily, badly, falling apart) and tap it with the scanner every hit once you think it's close. A toolbox on a ripley is about 3-5% and a wrench is about 1-2%.

**Comment by Exester509** [comment]
>you can already skip the 2nd mech simply by using a Ripley and upgrading it.

Exactly! It's unnecessary pointless tedium. If you're going to research combat mechs, having to scan one or two mechs will only change how long the yawn will be when you realize you gotta do the experiment. Considering robos get two exofabs, it's not even a timelock. This is a great QoL improvement.

File: code/modules/experisci/experiment/experiments.dm [hunk 1]
```diff
@@ -331,13 +331,13 @@
 
 /datum/experiment/scanning/random/mecha_damage_scan/final_contributing_index_checks(atom/target, typepath)
 	var/found_percent = round((target.get_integrity() / target.max_integrity) * 100)
-	return ..() && (found_percent <= (damage_percent + 2) && found_percent >= (damage_percent - 2))
+	return ..() && (found_percent <= (damage_percent + 5) && found_percent >= (damage_percent - 5))
```
Comment (Ghommie) [inline code/modules/experisci/experiment/experiments.dm:334]:
```suggestion
	return ..() && ISINRANGE(found_percent, damage_percent - 5, damage_percent + 5)
```

**Pull request merged**